quickstarts/aws/amazon-chatbot/config.yml

```diff
@@ -0,0 +1,31 @@
+slug: aws-chatbot
+description: |-
+  ## What is Amazon Chatbot?
```

Comment:
Looks like we are using "Amazon Chatbot" and "AWS Chatbot" interchangeably here, and looks like the product name is "AWS chatbot". Lets standardize on "AWS Chatbot" in the content here and also update the data source to use the same naming

Reply:
@MichelLosier, addressed, i have corrected the product name.